src/themes/teams/siteVariables.ts

```diff
@@ -63,6 +63,8 @@ export const timestampTextColor = gray04
 //
 // Fonts
 //
+export const fontSizeLarge = pxToRem(24)
+export const fontSizeMedium = pxToRem(18)
```

Comment:
a bit confused what is the difference between `medium` and `base`?

Reply:
base is smaller than medium - pairing the sizes from current Teams app so that I can provide the example accordingly

Reply:
will keep the default value of the font-size for now to keep the changes to minimum